cl/corpus_importer/management/commands/scrape_pacer_free_opinions.py

```diff
@@ -128,14 +128,14 @@ def get_and_save_free_document_reports(options):
                              pacer_court_id, next_start_d, next_end_d)
                 mark_court_done_on_date(PACERFreeDocumentLog.SCRAPE_FAILED,
                                         pacer_court_id, next_end_d)
-                break
+                continue
```

Comment:
Could this be a bad merge? I recently changed this to be a break, not a continue? Same with line 138 below too.

Reply:
Yes. Bad merge from my debugging.